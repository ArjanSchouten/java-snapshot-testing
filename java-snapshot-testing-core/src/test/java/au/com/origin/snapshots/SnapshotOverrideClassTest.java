package au.com.origin.snapshots;

import static au.com.origin.snapshots.SnapshotMatcher.start;
import static au.com.origin.snapshots.SnapshotMatcher.validateSnapshots;

import au.com.origin.snapshots.config.BaseSnapshotConfig;
import org.junit.jupiter.api.AfterAll;
import org.junit.jupiter.api.BeforeAll;

public class SnapshotOverrideClassTest extends SnapshotSuperClassTest {

  @BeforeAll
  static void beforeAll() {
    start(new BaseSnapshotConfig());
  }

  @AfterAll
  static void afterAll() {
    validateSnapshots();
  }

  @Override
  public String getName() {
    return "anyName";
  }
}
