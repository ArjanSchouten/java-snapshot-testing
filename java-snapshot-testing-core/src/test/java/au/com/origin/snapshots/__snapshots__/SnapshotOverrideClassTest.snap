au.com.origin.snapshots.SnapshotOverrideClassTest.beforeAll=[
  "anyName"
]


au.com.origin.snapshots.SnapshotOverrideClassTest.shouldMatchSnapshotOne=[
  "anyName"
]